k8s/calibrate/prod-calibrate-api.yaml

```diff
@@ -26,13 +26,7 @@ spec:
         - name: calibrate-pickle
           image: us.gcr.io/airqo-250220/airqo-calibrate-pickle-file:latest
           imagePullPolicy: Always
-          resources:
```

Comment:
These resource specifications for the init container resource specifications were stopping the main container from starting